draft-lenders-dns-over-coap.md

```diff
@@ -265,6 +265,17 @@ the server.
 CoAP/CoRE Integration
 =====================
 
+DoC server considerations
+-------------------------
+To reduce the number of message exchanges within an LLN, a DoC server MAY try
+to resolve CNAME chains received in DNS responses from an upstream DNS resolver.
+
+If not explicitly requested in the DNS query, it is RECOMMENDED to strip all record entries of type
+NS and class IN and all associated A and AAAA records from DNS responses received from upstream DNS
+resolvers.
```

Comment:
I wonder if we can generalize that, so that we remove the `authority` and `additional` sections (if present), but keep the `answer` section. (See here)[https://datatracker.ietf.org/doc/html/rfc1035#section-4.1].  This way, we would not need to explicitly mention the types to strip.

Reply:
we don't need to wonder, we need to read the RFCs, which i'm currently doing.